### DISCUSSION TITLE
I would like to have an option to indicate the difficulty level in numbers

### DISCUSSION THREAD

**Discussion by vadrile** [ppy/osu#24266] [Ideas] [answered]
![2023y07m17d_132904813](https://github.com/ppy/osu/assets/139732508/0a9acb6c-0392-431e-a7bc-314df3c1b018)

The more stars there are, the more difficult it becomes to distinguish at a glance, so I think it would be easier to understand if it could be displayed in numbers
I'm sorry if the English is strange because I use machine translation.

**Comment by peppy** [comment, marked as answer]
You'll be glad to know this is fixed in an upcoming design refresh

![2023-07-17 14 00 25@2x](https://github.com/ppy/osu/assets/191335/b26c199b-20da-4412-858b-871358d61f24)

**Reply by vadrile** [reply]
oh, exactly the ideal change!
Looking forward to the release XD
